x-pack/plugins/canvas/canvas_plugin_src/functions/server/src/demodata/index.js

```diff
@@ -12,15 +13,20 @@ export const demodata = () => ({
   name: 'demodata',
   aliases: [],
   type: 'datatable',
-  help: 'A mock data set that includes project CI times with usernames, countries and run phases',
+  help: i18n.translate('xpack.canvas.functions.demodataHelpText', {
```

Comment:
please add .demodata. namespace for all ids in this file.
also use .args. namespace